suite-native/biometrics/src/components/BiometricOverlay.tsx

```diff
@@ -13,10 +16,26 @@ const overlayWrapperStyle = prepareNativeStyle(utils => ({
 
 export const BiometricOverlay = () => {
     const { applyStyle } = useNativeStyles();
+    const { translate } = useTranslate();
+    const { isBiometricsAuthenticationCanceled, setIsBiometricsAuthenticationCanceled } =
+        useIsBiometricsAuthenticationCanceled();
+
+    const handleReenable = () => {
+        // Setting this to true lets useBiometrics to ask for biometrics in case it was canceled by user before
+        // https://github.com/trezor/trezor-suite/issues/10647
+        setIsBiometricsAuthenticationCanceled(false);
+    };
```

Comment:
Can this be done automatically in the useBiometrics hook?

Reply:
The problem now is neverending flow of biometrics auth requests. I needed to stop that. That can be done by catching `user_cancel` error and stopping auth. 

This button is the way to allow it again somehow. 

Only other way I came with would be adding a delay before the next auth. But how long? And how to reenable earlier? 

This is where the button comes handy, because it is up to user to resolve this, since he was the one canceling the auth previously by tapping the cancel button.